### PR TITLE
History

### PR DESCRIPTION
**Pull request by nicoleqi7**
I added user info schema when user signup. waiting for real user data(calories & user info)

### CODE REVIEW

File: client/src/components/ProTip.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function ProTip() {
     <Typography sx={{ mt: 6, mb: 3 }} color="text.secondary">
       <LightBulbIcon sx={{ mr: 1, verticalAlign: 'middle' }} />
       Pro tip: See more <Link href="https://mui.com/getting-started/templates/">templates</Link> on
-      the MUI documentation.
+      the MUI documentation.adsasddasdasadas
```
Comment (RetroInferno) [inline client/src/components/ProTip.tsx:19]: lol, What's this good stuff? `adsasddasdasadas`

**Review by ahmadalmomani96 — CHANGES_REQUESTED** [review]
Overall, the code you've written is solid. You will have to go back and revisit code to make sure it's passing all of the tsx tests. 

Regards, 

Ahmad Almomani

**Review by RetroInferno — APPROVED** [review]
Aesome!

**Review by ahmadalmomani96 — APPROVED** [review]

**Comment by ahmadalmomani96** [comment]
Great work Nicole

**Pull request merged**